### PR TITLE
Convert over to `go mod` and add a Makefile

### PR DESCRIPTION
**Pull request by joestump**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
+
+deps:
+	go mod tidy
+	go mod vendor
+build:
+	GOARCH=amd64 GOOS=linux go build -o heroku-cloudwatch-drain 
+
+.PHONY: deps build
```
Comment (kiranpraneeth) [inline Makefile]: add make clean and make all too to start with? We can add make rebuild, make linux and make test later

**Comment by lifangmoler** [comment]
After I read what is `go mod` - `an alternative to GOPATH`.
Any conflicts with existing services under GOPATH, or planning to convert all of them to `go mod` also? 

I noticed 500+ lines vendor files. I do not have strong options about if we should commit them since everything in vendor/ is just text, so there's very little memory cost.

**Comment by joestump** [comment]
@lifangmoler this is a one-off app that won't touch anything outside of itself. Just a place for Heroku to shovel logs.

File: Makefile (hunk 1 above)
Comment (joestump) [reply Makefile]: Can we ticket these for later? Just trying to get centralized logs up and running so we can debug the event issue.

**Pull request merged**